### PR TITLE
s3: update Scaleway's configuration options - fixes #7507

### LINKED ISSUE #7507: Allowing "crypt" drives to reside in many other drives (not just 1 drive)
#### What is your current rclone version (output from `rclone version`)?

```
rclone v1.65.0
- os/version: ubuntu 20.04 (64 bit)
- os/kernel: 5.15.0-89-generic (x86_64)
- os/type: linux
- os/arch: amd64
- go/version: go1.21.4
- go/linking: static
- go/tags: none
```

#### What problem are you are trying to solve?

Link multiple small drives and limit them to 1 larger drive using rclone's crypt drive function

#### How do you think rclone should be changed to solve that?

https://rclone.org/crypt/

I have an idea that rclone can create additional features of crypt drives, for example allowing crypt drives to reside in many other drives (not just 1 drive). Example: we have 5 mega.io accounts of 20GB, 3 Google Drive accounts of 15GB... they can combine them into (5 x 20GB) + (3 x 15GB) ~ 145GB of "crypt" drive space.
When uploading a file to a "virtual" crypt drive, it will divide the file into files for example 1MB, 10MB... written to "real" drives (5 mega.io accounts of 20GB, 3 accounts). Google Drive account 15GB...)
When the user needs to download, rclone will download small files (eg 1MB, 10MB...) that have been divided and encrypted, then reassemble and create the original file.
I think it's possible, it's pretty magical, it looks like we're creating a universe inside the little free storage packages of the storage providers.

#### How to use GitHub

* Please use the 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to show that you are affected by the same issue.
* Please don't comment if you have no relevant information to add. It's just extra noise for everyone subscribed to this issue.
* Subscribe to receive notifications on status change and new comments.

### PR DESCRIPTION
**Pull request by lavigne958**
#### What is the purpose of this change?

In order to handle special character, the configuration must specify rclone configuration to use `list_url_encode`.

Scaleway's S3 backend is configured to return the error 'BucketAlreadyOwnedByYou' when trying to create a new bucket you already own, in that matter set rclone to expect suche error using `use_already_exists`

Set upload size value to recommended values. 

#### Was the change discussed in an issue or in the forum before?

closes #7507 

#### Checklist

- [x] I have read the [contribution guidelines](https://github.com/rclone/rclone/blob/master/CONTRIBUTING.md#submitting-a-new-feature-or-bug-fix).
- [x] I have added tests for all changes in this PR if appropriate.
- [x] I have added documentation for the changes if appropriate.
- [x] All commit messages are in [house style](https://github.com/rclone/rclone/blob/master/CONTRIBUTING.md#commit-messages).
- [x] I'm done, this Pull Request is ready for review :-)

### CODE REVIEW

**Comment by albertony** [comment]
The MANUAL.* and rclone.1 files are autogenerated, so you should revert your changes in those files. Thanks.

**Comment by lavigne958** [comment]
> The MANUAL.* and rclone.1 files are autogenerated, so you should revert your changes in those files. Thanks.

alright will do, where is the source of the content then please ? :thinking:

File: docs/content/s3.md [hunk 1]
```diff
@@ -3359,10 +3359,13 @@ endpoint = s3.nl-ams.scw.cloud
...
 region = nl-ams
-location_constraint =
+location_constraint = nl-ams
 acl = private
-server_side_encryption =
-storage_class =
+upload_cutoff = 5M
+chunk_size = 5M
+copy_cutoff = 5M
+list_url_encode = true
```
Comment (ncw) [inline docs/content/s3.md:3367]: You don't need to put list_url_encode or use_already_exists here as they are set in a quirk now - I'd remove them from here

**Review by ncw — COMMENTED** [review]
This looks great thank you.

Please see minor comment inline!

Thank you

File: docs/content/s3.md (hunk 1 above)
Comment (lavigne958) [reply docs/content/s3.md]: true. done :heavy_check_mark:

**Review by ncw — APPROVED** [review]
This is great now - thank you :-)

I'll merge now.

**Comment by ncw** [comment]
Thank you

**Pull request merged**